articles/case_studies/fec-javascript-testing.md

```diff
@@ -0,0 +1,70 @@
+# Testing and refactoring JavaScript with FEC
+
+Until August of this year, the JavaScript for the OpenFEC project had no unit tests. This wasn't because the development team wasn't interested in testing--our backend code was already thoroughly tested. Instead, our JavaScript code just wasn't pleasant to test. Here's an example of the coding style that prevailed not long ago:
```

Comment:
perhaps you should specify that it is "front end JavaScript" since you differentiate between "JavaScript" and "backend code", which on the first read was a bit confusing for me